docs/content/dagster-cloud/developing-testing/dagster-cloud-yaml.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Confiuguration options for dagster_cloud.yaml | Dagster Docs
+description: How to use dagster_cloud.yaml to configure Dagster Cloud
+---
+
+# Dagster_cloud.yaml Guide
+
+<Note>This guide is applicable to Dagster Cloud.</Note>
+
+This dagster_cloud.yaml file is used to configure a Dagster Cloud instance. The file is required to tell Dagster where the code sources are to load into Dagster Cloud but has other options as well. For Hybrid Deployments, secrets are configured in this file as well.
```

Comment:
I wouldn't describe this is as "configure a Dagster Cloud instance" - it's specifically configuring how a code location should be built and deployed to Dagster Cloud usually as part of CI/CD. The reason secrets are configured is because those secrets are part of the code location.